modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -6,53 +6,236 @@
+
+Gibts schon was: https://doc.owncloud.com/server/admin_manual/configuration/server/config_apps_sample_php_parameters.html#openid-connectoidc-configuration
+
+Below you find a list of all `config.php` parameters available to configure OpenID Connect on ownCloud Server.
+
+- `provider-url` - the URL of the Identity Provider (for some Identity Providers like Keycloak or Azure AD this parameter holds more then just a domain but also a path)
+- `client-id` - the client ID for the ownCloud Web interface as specified in the Identity Provider client registration
+- `client-secret` - the client secret for the ownCloud Web interface as specified in the Identity Provider client registration
+- `mode` - the mode to search for users in ownCloud - possible values are `userid` or `email`
+- `search-attribute` - the attribute which is taken from the access token JWT or user info endpoint to identify a user (?? names as claim as well ??)
+- `loginButtonName` - by default the OpenID Connect App will add a button on the login page that will redirect the user to the Identity Provider and allow authentication via OIDC. This parameter allows to modify the button text.
+- `autoRedirectOnLoginPage` - if set to `true`, the ownCloud login page will redirect directly to the Identity Provider login without requiring the user to click a button.
+- `redirect-url` - the full URL under which the ownCloud OpenID Connect redirect url is reachable (only needed in special setups)
+- `insecure` - if set to `true`, no ssl verification will take place when connecting to the Identity Provider (DON'T use this in production)
+- `scopes` - depending on the setup and needs of the Identity Provider, the list of required scopes is entered here
+- `provider-params` - additional configuration depending on the Identity Provider can be entered here (usually only necessary if the Identity Provider does not support service discovery)
+- `auth-params` - additional parameters which are sent to the Identity Provider during the auth requests
+- `use-token-introspection-endpoint` - if set to `true`, the token introspection endpoint is used to verify a given access token (only needed if the access token is not a JWT)
+- `token-introspection-endpoint-client-id` - the client id to be used with the token introspection endpoint
+- `token-introspection-endpoint-client-secret` - the client secret to be used with the token introspection endpoint
+- `post_logout_redirect_uri` - a given URL where the Identity Provider should redirect to after logout
+- `allowed-user-backends` - use this parameter to limit the users which are allowed to login to a specific user backend (e.g., LDAP) if you want to prevent local db users to log in without OpenID Connect => VERIFY + EXAMPLE
+- `use-access-token-payload-for-user-info` - if set to `true`, any user information will be read from the access token. If set to `false`, the userinfo endpoint is used (requires version >= 1.1.0)
+
+=> needs table!
```

Comment:
Nice parameter table:

```suggestion
[cols=2,options=header]
|===
| Parameters
| Description

| `provider-url`
| the URL of the Identity Provider (for some Identity Providers like Keycloak or Azure AD this parameter holds more then just a domain but also a path)

| `client-id`
| the client ID for the ownCloud Web interface as specified in the Identity Provider client registration

| `client-secret`
| the client secret for the ownCloud Web interface as specified in the Identity Provider client registration

| `mode`
| the mode to search for users in ownCloud - possible values are `userid` or `email`

| `search-attribute`
| the attribute which is taken from the access token JWT or user info endpoint to identify a user (?? names as claim as well ??)

| `loginButtonName`
| by default the OpenID Connect App will add a button on the login page that will redirect the user to the Identity Provider and allow authentication via OIDC. This parameter allows to modify the button text.

| `autoRedirectOnLoginPage`
| if set to `true`, the ownCloud login page will redirect directly to the Identity Provider login without requiring the user to click a button.

| `redirect-url`
| the full URL under which the ownCloud OpenID Connect redirect url is reachable (only needed in special setups)

| `insecure`
| if set to `true`, no ssl verification will take place when connecting to the Identity Provider (DON'T use this in production)

| `scopes`
| depending on the setup and needs of the Identity Provider, the list of required scopes is entered here

| `provider-params`
| additional configuration depending on the Identity Provider can be entered here (usually only necessary if the Identity Provider does not support service discovery)

| `auth-params`
| additional parameters which are sent to the Identity Provider during the auth requests

| `use-token-introspection-endpoint`
| if set to `true`, the token introspection endpoint is used to verify a given access token (only needed if the access token is not a JWT)

| `token-introspection-endpoint-client-id`
| the client id to be used with the token introspection endpoint

| `token-introspection-endpoint-client-secret`
| the client secret to be used with the token introspection endpoint

| `post_logout_redirect_uri`
| a given URL where the Identity Provider should redirect to after logout

| `allowed-user-backends`
| use this parameter to limit the users which are allowed to login to a specific user backend (e.g., LDAP) if you want to prevent local db users to log in without OpenID Connect => VERIFY + EXAMPLE

| `use-access-token-payload-for-user-info`
| if set to `true`, any user information will be read from the access token. If set to `false`, the userinfo endpoint is used (requires version >= 1.1.0)
|===
```